src/openrct2-ui/windows/Viewport.cpp

```diff
@@ -35,23 +35,22 @@ namespace OpenRCT2::Ui::Windows
 
 #pragma region MEASUREMENTS
 
-    static constexpr StringId WINDOW_TITLE = STR_VIEWPORT_NO;
-    static constexpr int32_t WW = 200;
-    static constexpr int32_t WH = 200;
+    static constexpr StringId kWindowTitle = STR_VIEWPORT_NO;
+    static constexpr ScreenSize kWindowSize = { 200, 200 };
 
     static constexpr ScreenSize VIEWPORT_BUTTON = { 24, 24 };
 
 #pragma endregion
 
     // clang-format off
     static constexpr auto _viewportWidgets = makeWidgets(
-        makeWindowShim(WINDOW_TITLE, { WW, WH }),
-        makeWidget({      0, 14}, { WW - 1, WH - 1}, WidgetType::resize,   WindowColour::secondary                                         ), // resize
-        makeWidget({      3, 17}, {WW - 26, WH - 3}, WidgetType::viewport, WindowColour::primary                                           ), // viewport
-        makeWidget({WW - 25, 17}, VIEWPORT_BUTTON,   WidgetType::flatBtn,  WindowColour::primary  , ImageId(SPR_G2_ZOOM_IN),  STR_ZOOM_IN_TIP       ), // zoom in
-        makeWidget({WW - 25, 41}, VIEWPORT_BUTTON,   WidgetType::flatBtn,  WindowColour::primary  , ImageId(SPR_G2_ZOOM_OUT), STR_ZOOM_OUT_TIP      ), // zoom out
-        makeWidget({WW - 25, 65}, VIEWPORT_BUTTON,   WidgetType::flatBtn,  WindowColour::primary  , ImageId(SPR_LOCATE),      STR_LOCATE_SUBJECT_TIP), // locate
-        makeWidget({WW - 25, 89}, VIEWPORT_BUTTON,   WidgetType::flatBtn,  WindowColour::primary  , ImageId(SPR_ROTATE_ARROW),STR_LOCATE_SUBJECT_TIP)  // rotate
+        makeWindowShim(kWindowTitle, kWindowSize),
+        makeWidget({      0, 14}, { kWindowSize.width - 1, kWindowSize.height - 1}, WidgetType::resize,   WindowColour::secondary                                         ), // resize
+        makeWidget({      3, 17}, {kWindowSize.width - 26, kWindowSize.height - 3}, WidgetType::viewport, WindowColour::primary                                           ), // viewport
```

Comment:
I hope we can leverage some of the ScreenSize operators here:
```suggestion
        makeWidget({      0, 14}, kWindowSize - { 1, 1 }, WidgetType::resize,   WindowColour::secondary                                         ), // resize
        makeWidget({      3, 17}, kWindowSize - { 26, 3 }, WidgetType::viewport, WindowColour::primary                                           ), // viewport
```